### DISCUSSION TITLE
Remove ALL info

### DISCUSSION THREAD

**Discussion by TuriusUSA** [sherlock-project/sherlock#1930] [Q&A] [answered]
So, I used this program and like it...
I would like to know the easiest way to remove ALL of the info from the sites listed from the search...

Is there a way to remove ALL info from these sites?

Thank you

Rev. Ken

**Comment by ppfeister** [comment, marked as answer]
For future readers, closing as not relevant to Sherlock

Sherlock has no control over what data websites retain or make available. Contact the websites in question for removal.
